processor-steps/src/main/java/boringyuri/processor/common/steps/CommonInternalAnnotations.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2022 Anton Novikau
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package boringyuri.processor.common.steps
+
+import boringyuri.api.*
+import boringyuri.api.adapter.TypeAdapter
+import boringyuri.api.constant.*
+import boringyuri.api.matcher.MatcherCode
+import boringyuri.api.matcher.MatchesTo
+import boringyuri.api.matcher.WithUriMatcher
+import com.squareup.javapoet.ClassName
+import com.squareup.javapoet.TypeName
+
+internal val COMMON_INTERNAL_ANNOTATIONS: Set<TypeName> = hashSetOf(
```

Comment:
nit: I guess these are not just common internal annotations, but factory internal annotations as this set of annotations makes sense only for `UriFactoryProcessor`

Reply:
done